ui/src/sentry.ts

```diff
@@ -4,7 +4,7 @@ import { Integrations } from '@sentry/tracing'
 import { sentryEnabled } from '@lib/utils/sentryHelpers'
 
 if (sentryEnabled) {
-  const version = process.env.REACT_APP_VERSION
+  const version = process.env.RELEASE_VERSION
   const name = process.env.REACT_APP_NAME
   const hash = process.env.REACT_APP_COMMIT_HASH
   const release = `${name}@${version}+${hash}`
```

Comment:
I think you can substitute the `release` with the version totally.

Reply:
Now `RELEASE_VERSION ` is the actual version of the dashboard, and `REACT_APP_VERSION ` is the suffix of LocalStorage. Is this good?

Reply:
Yes, I think it's fine, since in sentry we just want a version as well.